docs/api.md

```diff
@@ -130,7 +132,7 @@ Note this corresponds to the format of the referenceMetadata seen in [SS4](#ss-4
 {
   *[statedInPropId]: [externalIdItem],
   *[externalIdProp]: [externalIdVal],
-  *[referenceUrl]: [String],
+  *[referenceUrl]: [String], // This is optional
   [dateRetrieved]: [String]
```

Comment:
Would it make sense to format the date in the way needed by the api here rather than using a string?
See: https://github.com/wmde/reference-island/blob/master/wikidata_game/api.php#L45

Reply:
I would say no, as it couples it too closely to the game (and we've already established that we might want to create different outputs of the reference dump, such as quickstatements)